### ISSUE TITLE
Make latest android APK available from alephium.org directly

### ISSUE THREAD

**Issue by diomark** [alephium/alephium-frontend#90]
Is it possible to make the android wallet APK available directly outside of the play store? Some phone manufacturers don't support the play store, and also this would allow updating to the latest version without having to wait for google to release the updated apk.

**Comment by nop33** [comment]
@mvaivre mentioned that:

> I’ve looked around a bit: metamask & blue wallet provide the APK on github, Rainbow does not.
> I’ve just found a [preview feature](https://docs.expo.dev/build/building-from-github/) to build on Github! This may be promising, so we can look into it and see if it’s straightforward or not.

**Comment by nop33** [comment]
Research on other popular mobile wallet apps:

> TL;DR: Out of some popular mobile wallets, only Metamask and Trust Wallet allow for APK download. OTOH, Uniswap, Rainbow, Ledger Live, and Coinbase Wallet don't.

## Metamask
✅  Provides APK download link through GitHub releases:
- I can't find the specific workflow that builds the APK that gets uploaded to the release (here is a search of [all workflows that requre "write" access](https://github.com/search?q=repo%3AMetaMask%2Fmetamask-mobile+%22contents%3A+write%22&type=code) to the GitHub releases) but I can find these:
- https://github.com/MetaMask/metamask-mobile/blob/main/.github/workflows/build-android-e2e.yml
- https://github.com/MetaMask/metamask-mobile/releases/tag/v7.53.0
- https://github.com/MetaMask/metamask-mobile/blob/main/.github/workflows/create-release-draft.yml
- https://github.com/MetaMask/metamask-mobile/blob/main/scripts/create-release-draft.sh

<img width="847" height="419" alt="Image" src="https://github.com/user-attachments/assets/a77ffdca-0dad-474a-b50a-9c260f6846df" />

## Uniswap
:x: Doesn't provide APK

<img width="691" height="336" alt="Image" src="https://github.com/user-attachments/assets/de93410d-3332-4846-93b1-a0a23d1081dc" />

## Trust wallet
✅ Allows for APK download

<img width="1022" height="857" alt="Image" src="https://github.com/user-attachments/assets/cbdd7ade-5ad2-40bd-b1de-c5e3ff5c41a9" />

## Rainbow wallet

:x: Doesn't provide APK

<img width="1432" height="647" alt="Image" src="https://github.com/user-attachments/assets/f8e69303-4dec-4f5d-852b-c1027048743e" />

## Ledger Live

:x: Also doesn't

<img width="1424" height="687" alt="Image" src="https://github.com/user-attachments/assets/4c2ade63-898c-4df1-bda0-c58bf4213c2a" />

## Coinbase Wallet

:x: Also doesn't

<img width="1256" height="565" alt="Image" src="https://github.com/user-attachments/assets/5b90525b-8260-4047-97a9-98065a1f2f7f" />
